.github/workflows/performance.yml

```diff
@@ -0,0 +1,208 @@
+name: Performance Tests
+
+on:
+  push:
+    branches:
+      - trunk
+      - '6.[2-9]'
+      - '[7-9].[0-9]'
+    tags:
+      - '[6-9]+.[2-9]'
+      - '[6-9]+.[2-9].[0-9]+'
```

Comment:
I think our branch filters are now correct. But a few more tweaks on the tag filtering.

This will match all version tags 6.2-6.9, but then only matches 7.2-7.9, 8.2-8.9, etc.

Let's just go with this:

```
      - '[0-9]+.[0-9]'
      - '[0-9]+.[0-9].[0-9]+'
      - '![3-5].[0-9].[0-9]+'
      - '!6.[01].[0-9]+'

```

This will:
- include all `X.Y` version tags (we know there will never be another `6.0`, `4.5`, etc.)
- include all `X.Y.Z` version tags
- Selectively exclude all `X.Y.Z` versions that start with `3.`, `4.`, or `5.` while allowing `34.Y.Z`, etc. versions to work (that would be a wild version to reach, but if we are ever able to switch to more rapid releases, it could happen)
- Selectively exclude all `X.Y.Z` versions that start with `6.0.` or `6.1.` while allowing `60.Y.Z`

Reply:
Of cousre! I think @mukeshpanchal27 had this right and I got too cute trying to simplify it 🙃

Reply:
This is now fixed 🤞🏻